audio_separator/separator/architectures/vr_separator.py

```
@@ -297,32 +431,43 @@ def _execute(X_mag_pad, roi_size):
             with torch.no_grad():
                 mask = []
 
                 for i in tqdm(range(0, patches, self.batch_size)):
-
                     X_batch = X_dataset[i : i + self.batch_size]
                     X_batch = torch.from_numpy(X_batch).to(device)
                     pred = self.model_run.predict_mask(X_batch)
                     if not pred.size()[3] > 0:
-                        raise ValueError(f"Window size error: h1_shape[3] must be greater than h2_shape[3]")
+                        raise ValueError(
+                            f"Window size error: h1_shape[3] must be greater than h2_shape[3]"
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)